mobile-app/cypress/e2e/functional/wallet/dex/compositeswap.spec.ts

```diff
@@ -173,8 +176,29 @@ context("Wallet - DEX - Instant/Future Swap - tabs and dropdowns", () => {
     );
   });
 
+  // the value of price rates keeps changing - can't assert an absolute value
```

Comment:
Could you try with DFI -> dETH? So the difference is big?
<img width="360" alt="image" src="https://user-images.githubusercontent.com/13292662/189674379-f209bba2-371a-4b6b-88e6-9030bbda2b0a.png">
 
 The test for the value is crucial or the price rate bug could recur in the future. If absolute value isn't possible, we can make use of `checkValueWithinRange`